internal/drivers/postgresql/postgresql.go

```diff
@@ -218,6 +227,17 @@ func (p *postgresqlDriver) Import(config internal.ImporterConfig) error {
 		if size > 0 {
 			if err := executeSQL(pending.String()); err != nil {
 				logger.Trace("offending sql: %s", pending.String())
+				//Post the offending sql to a file log. For large batch inserts, the offending SQL takes up too much space in the terminal
+				file, fileErr := os.Create("offend.txt")
+				if fileErr != nil {
+					fmt.Println(fileErr)
+
+				}
+
+				writer := bufio.NewWriter(file)
+				writer.WriteString(pending.String())
+				writer.Flush()
```

Comment:
```suggestion
				if err := os.WriteFile("offend.txt", []byte(pending.String()), 0666); err != nil {
					p.logger.Error("unable to write offending sql to file: %w", err)
				}
```
you can use os.WriteFile